src/components/layout/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import Overlay from '@components/layout/Overlay/Overlay';
+import './Dialog.scss';
+import ButtonItem from '@components/common/ButtonItem/ButtonItem';
+import useLabels from '@hooks/useLabels';
+import { Dialog } from '@contexts/DialogContext';
+
+interface Props extends Dialog {
+    closeDialog: () => void;
+}
+
+export default function Dialog({ title, content, submitHandler, closeDialog }: Props) {
+    const labels = useLabels();
+
+    return (
+        <Overlay clickHandler={closeDialog}>
+            <dialog className="dialog">
+                <div className="dialog__close" onClick={closeDialog} title={labels.dialog.close}>
+                    <span></span>
+                    <span></span>
+                </div>
```

Comment:
close button seems to cover dialog title which may influence user experience - setting offline status is good example